<launch>
<arg name="x_pos" default="0.0"/>
<arg name="y_pos" default="0.0"/>
<arg name="z_pos" default="0.0"/>
<arg name="robot_num" default="0"/>

<param name= "robot_description" command="$(find xacro)/xacro --inorder $(find gazebo_test_pkg)/unit_box.urdf"/>
<param name = "box_$(arg robot_num)" value="unit_box_0"/> 
	<arg name ="init_pose1" value="-x $(arg x_pos) -y $(arg y_pos) -z $(arg z_pos) -R 0 -P 0 -Y 0"/>
	<node
		name="spawn_model"
		pkg="gazebo_ros"
		type="spawn_model"
		args="$(arg init_pose1) -param robot_description -urdf -model box_$(arg robot_num)"
		output="screen"/>
</launch>		